src/api/app/models/request_ignoration.rb

```diff
@@ -0,0 +1,8 @@
+class RequestIgnoration < ApplicationRecord
+  belongs_to :staging_workflow
+  belongs_to :bs_request, primary_key: :number
+
+  validates :staging_workflow, :bs_request, :description, presence: true
+  validates :bs_request_id, numericality: true, uniqueness: { scope: :staging_workflow_id, message: "'%{value}' is already ignored" }
+  validates :description, length: { maximum: 250 }
```

Comment:
why to 250? I would limit it to 255 or limit the database to 250. We are wasting some (little) space :cry:

Reply:
> why to 250? I would limit it to 255 or limit the database to 250. We are wasting some (little) space cry

Changed to 255.